src/entity.ts

```diff
@@ -768,7 +891,13 @@ export namespace entity {
    * @param {object[]} results The response array.
    * @param {object} results.entity An entity object.
    * @param {object} results.entity.key The entity's key.
-   * @returns {object[]}
+   * @param {object} typeCastOptions Config for custom `integerValue` cast.
```

Comment:
This is a comment about line dropping and brackets.

Reply:
Done